SU-SWS/stanford_profile
# D8CORE-457 and more

# READY FOR REVIEW

# Summary
- Lots of configuration settings and a new install module
- Moved config to config/sync due to config_installer issue
- Missing all sorts of tests

# Needed By (Date)
- ?

# Urgency
- Semi important

# Steps to Test

1. See steps 1-4 in https://github.com/SU-SWS/acsf-cardinalsites/pull/18
5. D8CORE-101: Review that the main nav is populated with items after install
6. D8CORE-148: Review that the installation has default content
7. D8CORE-148: Review that the default content is in a layout_builder template with sidenav (home page should be different but there is a missing feature in layout_builder so I've created a new ticket to look in to a work around.)
8. D8CORE-151: Review user roles and Howard/Lindsey's grants.
9. D8CORE-458: Review Card paragraph type for construction and aesthetics. 
10. D8CORE-456: Review banner paragraph type for construction and aesthetics
11. Review the rest of the configuration settings and code.

# Affected Projects or Products
- Lots

# Associated Issues and/or People
- D8CORE-101
- D8CORE-148
- D8CORE-151
- D8CORE-458
- D8CORE-457
- D8CORE-456

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
